deploy/kblib/templates/_storages.tpl

```diff
@@ -0,0 +1,13 @@
+{{/*
+Define component storages, including volumeClaimTemplates
+*/}}
+{{- define "kblib.componentStorages" }}
```

Comment:
make sure @ldming are aware of this change.

Reply:
nicely done!